archiver/direct.py

```diff
@@ -50,12 +50,24 @@ def __archive(self, submission: HcaSubmission):
                 self.__exchange_sample_and_project_accessions(submission, biosample_accessions, biostudies_accessions[0])
 
         if self.__ena_submitter:
-            self.__archive_ena_entities(ingest_entities_to_update, submission)
+            ena_accessions = self.__archive_ena_entities(ingest_entities_to_update, submission)
 
         for entity in ingest_entities_to_update:
             submission.add_accessions_to_attributes(entity)
             self.__updater.update_entity(entity)
 
+        accessions = self.__archive_accessions(biosample_accessions, biostudies_accessions, ena_accessions)
+
+        return accessions
+
+    @staticmethod
+    def __archive_accessions(biosample_accessions, biostudies_accession, ena_accessions):
+        return {
```

Comment:
No rush here, but I find named entities very helpful in making the code readable and compact.